ADAL/src/workplacejoin/mac/ADClientCertAuthHandler.m

```diff
@@ -38,36 +51,26 @@ + (void)resetHandler
 {
```

Comment:
From what I've seen though if the user takes too long to respond the connection will timeout and the whole window will get torn down anyways. Because the client TLS handshake is part of the connection itself it's subject to normal network timeouts, although from what I've seen in the past the preference is to not have things time out ever.